unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -164,6 +164,7 @@ static ERROR_MAP = {
   'pre_upload_error_empty_verb_limits': -51,
   'pre_upload_error_renaming_file' : -52,
   'pre_upload_error_fetch_redirect_url': -53,
+  'pre_upload_error_create_asset': -54,
```

Comment:
I have added an error for the access token which is -54, so there will be a conflict. I'm fine to switch to -55 or if you want to switch let me know. Either way works for me.